server/api/topics.js

```diff
@@ -67,10 +81,26 @@ router.post("/:topic_id/question", (req, res) => {
 	}
 	topicDb
 		.createNewQuestion(id, newQuestion)
-		.then(() => res.send("Question created!"))
-		.catch((err) => {
-			console.error(err);
-			res.json(500);
-		});
+		.then((data) => res.status(200).json(data))
+		.catch(() => {
+			res.status(500).json({
+				error: "creating a new question failed",
+			});
+		 });
+});
+router.put("/:topic_id/question", function (req, res) {
+	const newQuestion = req.body.question_text;
+	const id = req.params.topic_id;
+	if(!newQuestion){
+		return res.status(400).send("The question can not be empty");
```

Comment:
I suppose in here that can be consistent with the rest of the error responses 
```js
res.status(400).json({
  error: "The question can not be empty",
});
```

Reply:
I asked you about this and tell it is ok to write it like this

Reply:
do u want me to change it

Reply:
I think it's better. But if you are ok for with the way it is, I am merging